README.md

```diff
@@ -10,17 +10,18 @@ To run the TypeScript examples:
 ```
 $ npm install -g aws-cdk
 $ npm install
-$ npm build
-$ cdk deploy
+$ npm run build
+$ cdk synth   // Creates the Cloudformation template
```

Comment:
Not sure that we need to make them synth. It's not required, and presenting it like this makes it look like you have to do it.

Reply:
Fair. I like doing it just to see that it worked without having to actually spend the time deploying it -- and might be useful for those used to squinting at/cobbling together CFN templates. But I can make a note that it's optional.